tests/test_wave_01.py

```diff
@@ -143,12 +143,15 @@ def test_moves_movie_from_watchlist_to_watched():
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
     
-    raise Exception("Test needs to be completed.")
+    # raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
+    assert len(updated_data["watched"]) !=len(updated_data["watchlist"])
+    assert FANTASY_2 in updated_data["watched"]
+    assert HORROR_1 in updated_data["watched"]
+    assert FANTASY_1 in updated_data["watchlist"]
```

Comment:
Great! We discussed how to think about the assertions, and how the existing assertions were checking the lengths, but that we still needed to validate the movies in the lists were the ones we expected.